src/main/java/model/data/CombinedDataTable.java

```diff
@@ -123,6 +123,17 @@ public void deleteNotFlagged() {
 		}
 	}
 
+	@Override
+	public DataTable getTable(String name) {
+		if (table.getName().equals(name)) {
```

Comment:
Nee, joh. Het is misschien een beetje vaag, maar als een tabel uit meerdere subtabellen kan bestaan is het wel weer logisch. Kan prima.